package.json

```diff
@@ -97,7 +96,8 @@
     "redux-logger": "^3.0.6",
     "regenerator-runtime": "^0.10.3",
     "request": "^2.83.0",
-    "sass-loader": "^6.0.3",
+    "sass": "^1.64.2",
+    "sass-loader": "^7.3.1",
```

Comment:
Any reason you can't get higher than this version for `sass-loader`? It looks like the most recent is 13.3.2.